hwy/contrib/unroller/README.md

```diff
@@ -23,7 +23,7 @@ struct DoubleUnit : UnrollerUnit<DoubleUnit<int>, int, int> {
 Leaving all other methods in their default state, the following code will double all the values in array `a` and place them in `r`
```

Comment:
How about we add a comment that this interface is considered experimental and may still change? Then we can commit them already and gather practical experience with it, while still allowing later improvements.